posthog/models/feature_flag/permissions.py

```diff
@@ -12,7 +12,7 @@ def can_user_edit_feature_flag(request, feature_flag):
     else:
         if not request.user.organization.is_feature_available(AvailableFeature.ROLE_BASED_ACCESS):
             return True
-        if feature_flag.created_by == request.user:
+        if hasattr(feature_flag, "created_by") and feature_flag.created_by and feature_flag.created_by == request.user:
```

Comment:
Hmm, I don't think there's a case where `feature_flag.created_by` does not _exist_, it always has to be set to some value due to being on the model, right?

Reply:
it is needed, this is how django checks whether a foreign key exists without raising exceptions

<img width="926" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/f3eed9c2-1dbd-40fe-8338-e05af9633bf1">

and it would be annoying to catch in all of these places